azure_monitor/flask_sample/__init__.py

```diff
@@ -0,0 +1,29 @@
+import logging
+from logging import Logger
+from opencensus.ext.azure.log_exporter import AzureLogHandler
+from opencensus.trace import config_integration
+
+from .config import Config
+
+config_integration.trace_integrations(
```

Comment:
Please add inline comments for what this does.

Reply:
done. added comment